packages/teleport/src/cluster/components/Sessions/SessionList/sessionTableCells.test..tsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2020 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import NodeCell from './NodeCell';
+import DescCell from './DescCell';
+import cfg from 'teleport/config';
+import { render, Router } from 'design/utils/testing';
+
+const session = {
+  sid: undefined,
+  namespace: undefined,
+  login: '',
+  created: undefined,
+  durationText: undefined,
+  serverId: 'server-id',
+  hostname: undefined,
+  clusterId: undefined,
+  parties: undefined,
+  addr: undefined,
+};
+const data = [session];
+const tableRow = document.createElement('tr');
+
+test('empty hostname and addr check in NodeCell', () => {
```

Comment:
I think these unit-tests are too much for such a basic thing. 

1. Create 1 data sample with different cases (3 is enough)
2. Render the table and create a snapshot.